qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -1014,45 +726,18 @@ def get_num_channels(byte_rep: int) -> int:
             return 2
         elif byte_rep in four_channels:
             return 4
+        elif byte_rep in eight_channels:
+            return 8
         elif byte_rep in sixteen_channels:
             return 16
         else:
             raise RuntimeError('Invalid channel configuration supplied')
 
-
     def _read_register(self, offset: int) -> int:
```

Comment:
shall we not preserve the doc strings?

Reply:
They are moved to the api class from the driver class, so they are preserved. The thing is that register manipulation are private methods, hence direct driver users should not use them, and since the actual code of the methods is not in the api class, i decided to keep the docstrings there, and not copy them over to save the driver class from the details basically.

Is it ok? Or shall i write some docstrings here, or copy-paste docstrings (don't want that actually)?

Reply:
That is fine I think  :-)